aten/src/ATen/native/cudnn/RNN.cpp

```diff
@@ -600,6 +600,15 @@ namespace {
 #endif
   }
 
+  cudnnDataType_t promote_rnn_math_type(cudnnDataType_t dtype) {
+#if CUDNN_VERSION != 7103
```

Comment:
is it really `!=`? is it not something like `>=`? just checking

Reply:
yes this is supposed to be `!=`
In CUDNN 7.1.3 math type in rnn descriptor means something else and they have a check for it, so we are making an exception here.

Reply:
This is one of those cases where having a comment explaining that yes, it's intentional, helps future code readers ;)